guardrails/utils/telemetry_utils.py

```diff
@@ -100,6 +101,9 @@ def trace_validator_result(
         "instance_id": instance_id,
         **kwargs,
     }
+
+    TraceHandler().log_validator(validator_log)
```

Comment:
This will only be called if metrics are enabled by the user.  Is this the intent?

Reply:
When metrics are disabled it only stops the _sending_ of the traces.  With enable_metrics=false in my config file I still see metrics being populated.  Should we have a different field for enable/disable of tracing?

Reply:
Gotcha, just tested this out because the old behaviour was that it wouldn't be called.  This was back before the anonymous metrics collection and otel was an optional dependency, so no otel -> no tracer -> no span.  Now you are correct that it is always called but when the user does not pass in a tracer it yields a `NonRecordingSpan`.  All good then!